backend/src/cli/parser.py

```
@@ -0,0 +1,67 @@
+"""JSON file parsing utilities."""
+
+import json
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+from src.cli.models import BoundingBox
+
+
+def parse_annotation_file(file_path: Path) -> Tuple[str, List[BoundingBox]]:
```

Ruff (0.12.2)

Use `tuple` instead of `Tuple` for type annotation

Replace with `tuple`

(UP006)